root/static/styles/extra/homepage.less

```diff
@@ -11,6 +11,45 @@
     }
 }
 
+#appDownload {
+    .section {
+        padding: 60px 0;
```

Comment:
Feedback on the forums was that the redesign requires too much scrolling, and 60px does feel a bit much to me here...even 30px feels like more than enough padding, and leaves the section very visible still.  What do you think?

Reply:
Done